File: WarehouseManagementWeb.Application/Dto/Output/ResourceReceipt/ResourceReceiptItemOutput.cs
namespace WarehouseManagementWeb.Application.Dto.Output.ResourceReceipt
{
    /// <summary>
    /// Класс выходной модели входящих ресурсов поступления.
    /// </summary>
    public class ResourceReceiptItemOutput
    {
        /// <summary>
        /// Id ресурса поступления.
        /// </summary>
        public int ResourceReceiptId { get; set; }

        /// <summary>
        /// Id ресурса.
        /// </summary>
        public int? ResourceId { get; set; }

        /// <summary>
        /// Наименование ресурса.
        /// </summary>
        public string? ResourceTitle { get; set; }

        /// <summary>
        /// Id единицы измерения.
        /// </summary>
        public int MeasureUnitId { get; set; }

        /// <summary>
        /// Наименование единицы измерения.
        /// </summary>
        public string? MeasureUnitTitle { get; set; }

        /// <summary>
        /// Количество ресурса.
        /// </summary>
        public int ResourceQuantity { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/warehouse/document-receipt/models/output/resource-receipt-item-output.ts
/**
 * Класс выходной модели входящих ресурсов поступления.
 */
export class ResourceReceiptItemOutput {
  /**
   * Id ресурса поступления.
   */
  resourceReceiptId: number = 0;

  /**
   * Id ресурса.
   */
  resourceId: number = 0;

  /**
   * Наименование ресурса.
   */
  resourceTitle: string = '';

  /**
   * Id единицы измерения.
   */
  measureUnitId: number = 0;

  /**
   * Наименование единицы измерения.
   */
  measureUnitTitle: string = '';

  /**
   * Количество ресурса.
   */
  resourceQuantity: number = 0;
}
